source/references/components/scroller.md

```diff
@@ -1,40 +1,39 @@
 ---
 title: <scroller>
 type: references
-group: Build-in Components
-order: 8.09
+order: 2.8
 version: 2.1
 ---
 
 # &lt;scroller&gt;
 
 <span class="weex-version">v0.6.1+</span>
 
-A scroller is a component in vertical direction which can have multiple child components in one column. If total height of its child components exceed the height of the scroller, the whole child components will be scrollable.
+Scroller is a component which can have multiple child components in one column. It supports both direcitons. If the content size of child components exceeds the frame of the scroller, the whole child components will be scrollable.
```

Comment:
Maybe the description should be more reasonable, please refer to [iOS UIScrollView](https://developer.apple.com/documentation/uikit/uiscrollview?language=objc#) or [Android ScrollView](https://developer.android.com/reference/android/widget/ScrollView.html)